### ISSUE TITLE
Unity client fails to connect on iOS (Updated for 0.14.5)

### ISSUE THREAD

**Issue by tonygiang** [colyseus/colyseus-unity3d#143]
Tested on the latest Unity client (0.14.4): When ran on an iOS device, the join room task returned by `Client.JoinOrCreate` always finish immediately with `Faulted` status. No connection attempt is received on server-side (running Colyseus 0.14.13).

When running inside Unity Editor on MacOS, Colyseus Unity client is still able to connect.

**Comment by tonygiang** [comment]
Just an update: I traced down exactly where the connection always fails on Unity iOS client: this line in `Colyseus.Client.CreateMatchMakeRequest`:
![2021-05-10 12_06_37-Window](https://user-images.githubusercontent.com/7429894/117608441-3fdc4180-b188-11eb-84a6-5c342ebaddbe.png)
`req.isNetworkError` will be true after this line and the function quits.

**Comment by tonygiang** [comment]
This issue still remains in the 0.14.5 client.

**Comment by tonygiang** [comment]
This turned out to be an inexplicable iOS-specific quirk. By all online documentation, setting NSAllowsArbitraryLoads in XCode's propertly list to true should allow all http connections, but for some reason, our builds refuse to acknowledge this property and we had to add our Colyseus server domain to the exception list. I'm just about fed up with Apple products. But anyway, this issue is not related to Colyseus. Closing.
